Automattic/wp-calypso
# Plans 2023: Add the featured badges to multiple plans

Related to https://github.com/Automattic/martech/issues/1492

## Proposed Changes

Adds the featured badges to multiple plans
- "popular" for premium
- "best for devs" for business
- "best for stores" for eCommerce

~This only applies to the top grid/table, not the comparison grid at the bottom. See the screenshots below for how that looks right now (not great on desktop view, where more than one featured plans sit side-by-side).~

This applies to both the top and bottom (comparison) tables.

**Notes:** The "your plan" badge is handled in another branch/PR (https://github.com/Automattic/wp-calypso/pull/73582).

## Media

### Top table

#### desktop
<img width="700" alt="Screenshot 2023-02-16 at 4 09 13 PM" src="https://user-images.githubusercontent.com/1705499/219388472-97020c73-2d69-454f-977f-eab773fd7626.png">

#### tablet
<img width="700" alt="Screenshot 2023-02-16 at 4 09 33 PM" src="https://user-images.githubusercontent.com/1705499/219388498-335615b3-9d83-4ecc-9276-1deed4b18e1a.png">

#### mobile

<img width="300" alt="Screenshot 2023-02-16 at 4 09 50 PM" src="https://user-images.githubusercontent.com/1705499/219388741-2aa7e8f1-3457-429e-b5e5-a2e4de444a7c.png">

### Comparison grid

#### desktop
<img width="700" alt="Screenshot 2023-02-16 at 4 10 16 PM" src="https://user-images.githubusercontent.com/1705499/219389747-6be0495e-b7b2-4ee6-90f3-3f6e6fba370c.png">

## Testing Instructions



* go to `/plans/[SITE]` with `?flags=onboarding/2023-pricing-grid` and confirm the changes
* got to `/start/plans` and confirm the same

## Pre-merge Checklist
